requirements/runtime.txt

```diff
@@ -123,8 +115,6 @@ rich==13.7.1
     # via
     #   sigstore
     #   twine
-secretstorage==3.3.3
```

Comment:
This is in the tree as well:
https://inspector.pypi.io/project/keyring/25.6.0/packages/d3/32/da7f44bcb1105d3e88a0b74ebdca50c59121d2ddf71c9e34ba47df7f3a56/keyring-25.6.0-py3-none-any.whl/keyring-25.6.0.dist-info/METADATA#line.17